semeio/communication/semeio_script.py

```diff
@@ -91,3 +91,17 @@ def _get_output_dir(self):
     @property
     def reporter(self):
         return self._reporter
+
+    @property
+    def output_dir(self):
+        return self._output_dir
+
+    @output_dir.setter
+    def output_dir(self, output_dir):
```

Comment:
Do we really need to expose this publicly? Could it not be part of the initialisation?

Reply:
> Do we really need to expose this publicly? Could it not be part of the initialisation?

Technically we could, but, because we are wrapping `ErtScript` that would be very hacky as we dont control the creation of the class when the workflow is run from `ert`.

Reply:
That is true. But since we are only using it from within `SemeioScript`, perhaps we could make the setter private? 🤔

Reply:
I`ve no problem with that solution